astronomer/providers/http/hooks/http.py

```diff
@@ -131,6 +132,7 @@ async def run(
                         self.log.exception("HTTP error with status: %s", e.status)
                         # In this case, the user probably made a mistake.
                         # Don't retry.
+                        await session.close()
```

Comment:
i have closed the connection here since there is a retry and I need the connection to be alive until retry is done and its also in infinite loop